src/System.Net.Security/tests/FunctionalTests/NegotiateStreamStreamToStreamTest.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace System.Net.Security.Tests
 {
-    [PlatformSpecific(TestPlatforms.Windows)] // NegotiateStream only supports client-side functionality on Unix
+    [PlatformSpecific(Xunit.PlatformID.Windows)] // NegotiateStream only supports client-side functionality on Unix
```

Comment:
Why are these changing back to `Xunit.PlatformID`? We renamed `PlatformID` to `TestPlatforms` and `PlatformID` is being deleted. See #12284.